python/lsst/fgcmcal/fgcmOutputProducts.py

```diff
@@ -163,6 +163,9 @@ def __init__(self, *, config=None):
         if not config.doReferenceCalibration:
             self.outputs.remove("fgcmOffsets")
 
+    def getSpatialBoundsConnections(self):
+        return ("fgcmPhotoCalibCatalog",)
```

Comment:
This should be `fgcmPhotoCalib`.  (Or is this the connection name as a class variable, or the `name` property of the connection?